conference-finder/src/App.css

```diff
@@ -20,3 +20,27 @@
 .App-link {
   color: #09d3ac;
 }
+
+.card {
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+}
+
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
If you're looking for the items to be in a column instead of using `flex-wrap` you can use `flex-direction: column;` have a play around with https://the-echoplex.net/flexyboxes/ `flex-wrap` is essentially asking you, do you want to keep everything on each line in the same line or not ( which is what `flex-wrap: wrap;` does ).

Reply:
Will keep that in mind for when we discuss the visuals with the others.